scripts/Dockerfile.devel

```diff
@@ -1,19 +1,22 @@
 FROM ceph/ceph:v15
 
+ARG GOROOT=/usr/local/go
```

Comment:
why GOROOT need to be an argument inside the container?

Reply:
`ARG` does not export the variable in the environment, but is a notation to provide a variable in the Dockerfile. This way `GOROOT` only needs to be set once, and can be used on the single `ENV` line.